database/dataGen.js

```diff
@@ -0,0 +1,49 @@
+const faker = require('faker');
+const fs = require('fs');
+
+const getRandomInt = (min, max) => {
+  min = Math.ceil(min);
```

Comment:
If you know that you'll pass in an int for min and max, to save some time perhaps you don't need to call Math.ceil and Math.floor.